src/tensorwaves/optimizer/callbacks.py

```diff
@@ -30,6 +31,50 @@ def finalize(self) -> None:
         self.__progress_bar.close()
 
 
+class YAMLSummary:
+    def __init__(
+        self,
+        filename: str,
+        step_size: int = 10,
+    ) -> None:
+        """Log fit parameters and the estimator value to a `tf.summary`.
+
+        The logs can be viewed with `TensorBoard
+        <https://www.tensorflow.org/tensorboard>`_ via:
+
+        .. code-block:: bash
+
+            tensorboard --logdir logs
+        """
+        self.__iteration = 0
+        self.__step_size = step_size
+        self.__stream = open(filename, "w")
+
+    def __call__(self, parameters: dict, estimator_value: float) -> None:
+        self.__iteration += 1
+        if self.__iteration % self.__step_size != 0:
+            return
+        output_dict = {
+            "Time": datetime.now(),
+            "Iteration": self.__iteration,
+            "EstimatorValue": float(estimator_value),
```

Comment:
It would be good if we would be able to pass the estimator label through as well. Maybe through the init?

Reply:
Have a look 171b6e5 (it can be reverted)

Reply:
Looks ok. But would be a bit inconvenient for appending  see https://github.com/ComPWA/tensorwaves/pull/154#discussion_r525941074